Automattic/jetpack
# Jetpack: register video-chapters beta feature. Render super basic control component

On this PR:

* Register the `videopress/video-chapters` beta feature and ...
* Create a super basic `<DetailsControl />` control component, ...
* Extend the Edit function of the current implementation to ...
* Render the `<DetailsControl />` component in the sidebar of the  VideoPress block ...
* ...considering it's a beta extension

Fixes https://github.com/Automattic/jetpack/issues/26171

#### Changes proposed in this Pull Request:

* Jetpack: register video-chapters beta feature. Render super basic control component

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Test locally, Simple sites, JN sites
* You will need to know how to enable/disable [beta extensions](https://github.com/Automattic/jetpack/tree/trunk/projects/plugins/jetpack/extensions/#beta-extensions).
* Confirm you don't see the new `Details` panel when beta extensions is not enabled


<img width="958" alt="Screen Shot 2022-09-12 at 13 44 25" src="https://user-images.githubusercontent.com/77539/189710387-4be26eec-8f63-49d0-802d-1516421bf96e.png">
